components/coding/studentPortal/Sidebar.tsx

```diff
@@ -150,6 +164,26 @@ export const Sidebar: React.FC<SidebarProps> = ({}: SidebarProps) => {
             </svg>
           }
         />
+        <SidebarItem
+          name={"Goals"}
+          notifications={dispatch(isGoalApproaching(goalApproaching))}
```

Comment:
you shouldn't be dispatching an action here - you want to pass the Sidebar component a boolean value on whether a goal is approaching

Reply:
changed to only goalApproaching, which is the boolean value